src/app/Http/Controllers/API/PollController.php

```diff
@@ -102,7 +129,7 @@ public function publish(Request $request, Poll $poll)
      */
     public function results(Request $request, Poll $poll)
     {
-
+        return PollResource::collection($poll->users;
```

Comment:
only presenter could show results + there is a missing `)`
Have you tested your code ?

Reply:
Fixed in b75458f